scripts/config-dhcp-server.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+#copy the configuration information to network-conf.txt
+echo "config-server" > /home/pi/network.conf
+
+#Bring down interface eth0
+sudo ifdown eth0
+echo "Interface eth0 is down"
+
+#Bring up eth0 with DHCP Server configuration
```

Comment:
shouldn't something happen here? like replacing or editing /etc/network/interfaces?

Reply:
So we are leveraging what I would deem an obscure feature/function of [interfaces(5)](https://manpages.debian.org/stretch/ifupdown/interfaces.5.en.html) using the 'mapping' feature which calls a script to determine the configuration for an interface. That script here is ./scripts/eth0-mapping-config.sh and it returns a string for the mapping.

The 'something to happen' that you are looking for is at line 4 above: `config-server`.

Reply:
ah, ok then!